app/game/memoryGame/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="script.js" defer></script>
+    <title>Memory Game</title>
+</head>
+<body>
+    <table id="container">
+        <tbody>
+            <tr>
+                <td><canvas class="canvas" id="canvas0"></canvas></td>
+                <td><canvas class="canvas" id="canvas1"></canvas></td>
+                <td><canvas class="canvas" id="canvas2"></canvas></td>
+            </tr>
+            <tr>
+                <td><canvas class="canvas" id="canvas3"></canvas></td>
+                <td><canvas class="canvas" id="canvas4"></canvas></td>
+                <td><canvas class="canvas" id="canvas5"></canvas></td>
+            </tr>
+            <tr>
+                <td><canvas class="canvas" id="canvas6"></canvas></td>
+                <td><canvas class="canvas" id="canvas7"></canvas></td>
+                <td><canvas class="canvas" id="canvas8"></canvas></td>
+            </tr>
+        </tbody>
```

Comment:
The grid is hard-coded as 3x3, which seems perfectly fine. I'd suggest adding elements these dynamically, but it's not necessary